test/passing/quoted_strings.ml

```diff
@@ -0,0 +1,65 @@
+let foo = {%foo | foooooooooooooo |}
+
+let foo =
+  ( {%foo | foooooooooooooo | (* B *) } (* C *))
+
+let foo = ({%foo| foooooooooooooo |}
+  [@attr] )
+
+let foo = (* A *) ({%foo | foooooooooooooo |} (* B *) [@attr]) (* C *)
+
+let foo = (* A *) {%foo sep| foooooooooooooo |sep}
+
+let foo = {%foo | foooooooooooooo |} [@@attr]
+
+let foo =
+  ( {%foo | foooooooooooooo | (* B *) } (* C *) [@@attr] (* D *))
+
+let foo = ({%foo| foooooooooooooo |}
+  [@attr] )
+  [@@attr]
+
+let foo = ({%foo | foooooooooooooo |} (* A *) [@attr]) (* B *) [@@attr]
+
+let foo = (* A *) ({%foo | foooooooooooooo |} [@attr]) (* B *) [@@attr]
+
+let foo = (* A *) {%foo sep| foooooooooooooo |sep} (* B *) [@@attr]
+
+{%%foo | foooooooooooooo |}
+
+{%%foo | foooooooooooooo | (* A *) } (* B *)
```

Comment:
Ah! I'm surprised this parses. What does A get attached to? the payload?

Reply:
Oh indeed, I just realized that the string is not closed at this line, it's closed at the `|}` in the next item, so the comments are not allowed here, I will clean up the tests.